plugins/audit_log/lib/samson_audit_log/audit_presenter.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module SamsonAuditLog
+  class AuditPresenter
+    ## Presenter for Audit Logger
+    class << self
+      AVAILABLE_PRESENTERS = [
+        :deploy,
+        :deploy_group,
+        :project,
+        :stage,
+        :user,
+        :user_project_role
+      ].freeze
+
+      def present(object)
+        return unless object
+        type = object.class.name.underscore.to_sym
+        if AVAILABLE_PRESENTERS.include?(type)
+          send(type, object)
+        else
+          object
```

Comment:
The logger should capture the serialization error silently.  Alternatively I could run a serialization here, because there are a few cases where I send over hash's as the object.